### DISCUSSION TITLE
Standard Notes for SailfishOS (SFOS)

### DISCUSSION THREAD

**Discussion by meolic** [standardnotes/forum#2746] [Suggestions]
Is it possible to get a native app for SailfishOS (SFOS)? It is a Linux + Qt-based ecosystem. I can help (e.g. testing) but I am not an experienced  SFOS developer.

https://sailfishos.org/wiki/SailfishOS

**Comment by JaspalSuri** [comment]
Hi @meolic, thank you for the suggestion. I'm afraid that since our development bandwidth can't be expanded to support too many additional platforms at the moment, SailFishOS users might need to use the [Android app](https://standardnotes.com/download) instead (if still possible).
